.github/renovate.json5

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
You may want to look into using GitHub's Merge Queue. I see you're not using it...

The main Renovate repository uses the Merge Queue, and it's made merging so much easier and faster. I don't need to update my branches when `main` changes under me after approval. The GitHub Merge Queue will auto-update branches, without making any more noise.

Read the [GitHub Docs, How merge queues work](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#how-merge-queues-work) to learn the specifics.

Read about [Renovate support for GitHub's Merge Queue](https://docs.renovatebot.com/key-concepts/automerge/#github-merge-queue).

You'll need to ask somebody else for help to get GitHub Actions configured correctly for the Merge Queue. 😄

Reply:
I don't want to hold up this PR by waiting for GitHub's Merge Queue to be enabled on this repository. So I'm opening a discussion instead:

- #17582

Can you please resolve this comment? 😉